src/include/main/jucc.h

```diff
@@ -1,15 +1,41 @@
 #ifndef JUCC_JUCC_H
 #define JUCC_JUCC_H
 
+#include <algorithm>
```

Comment:
why are unnecessary headers included?

Reply:
std::find
std::reverse

Reply:
include in .cpp file then .h doesn't require this.